charts/spire/charts/spire-server/values.yaml

```diff
@@ -87,6 +87,12 @@ trustDomain: example.org
 
 bundleConfigMap: spire-server
 
+federation:
+  enabled: false
+  bundleEndpoint:
+    port: 8443
+    address: "0.0.0.0"
```

Comment:
```suggestion
  enabled: false
  bundleEndpoint:
    port: 8443
    address: "0.0.0.0"
  federatesWith:
    trustDomain: broker.example.org
    bundleEndpoint:
      url: ""
  ingress:
    enabled: false
```

Reply:
please see this previous discussion:

https://github.com/spiffe/helm-charts/pull/97#issuecomment-1458401788

There are better ways to configure `federatesWith` in kubernetes. The scope of this change is just to enable the bundle endpoint.

Reply:
If we get the general plugin config stuff in place, then federatesWith could be added that way?